scripts/sonic_sku_create.py

```diff
@@ -0,0 +1,805 @@
+#! /usr/bin/env python
```

Comment:
>python [](start = 16, length = 6)

Please make sure the same script could be run by python3

Reply:
tabulate module used in the script does not work with python3. It says module not found.

Reply:
So if this is true, please explicitly use python2 in shebang line.

---
In reply to: [508805018](https://github.com/Azure/sonic-utilities/pull/1163#discussion_r508805018) [](ancestors = 508805018)

Reply:
Addressed in next commit